test/FactorReTerm.jl

```diff
@@ -30,7 +30,7 @@ const LMM = LinearMixedModel
     end
 
     @testset "utilities" begin
-        @test levs(sf) == string.('A':'F')
+        @test refpool(sf) == string.('A':'F')
```

Comment:
They're equivalent in our implementation, but `levels` (instead of `refpool`) is closer to the direct replacement to `levs`.

Reply:
Milan pointed out in another discussion that the value of `refpool` can include unused levels and `missing` whereas the value of `levels` cannot include `missing` and should drop unused levels.  I will change that here.